jdaviz/core/loaders/importers/image/image.py

```diff
@@ -39,6 +38,7 @@ class ImageImporter(BaseImporterToDataCollection):
     extension_items = List().tag(sync=True)
     extension_selected = Any().tag(sync=True)
     extension_multiselect = Bool(True).tag(sync=True)
+    gwcs_to_fits_sip = Bool(True).tag(sync=True)
```

Comment:
I think this will need to be exposed in the user-API in order to support passing through `load` (and eventually through `load_data` when @javerbukh builds on top of this)